test/controllers/api/v2/trends_controller_test.rb

```diff
@@ -1,68 +0,0 @@
-require 'test_helper'
-
-class Api::V2::TrendsControllerTest < ActionController::TestCase
-  def setup
-    @foreman_trend_valid_attrs = { :trendable_type => 'Model' }
-    @foreman_trend_invalid_attrs = { :trendable_type => 'NotExists' }
-
-    @foreman_trend = FactoryBot.create(:foreman_trend, :trendable_type => 'Environment', :fact_name => "fact")
-    @host = FactoryBot.create(:host)
-    FactoryBot.create(:fact_value, :value => '2.6.9', :host => @host,
-                      :fact_name => FactoryBot.create(:fact_name, :name => 'kernelversion'))
-    @fact_os = FactoryBot.create(:fact_value, :value => 'fedora', :host => @host,
-                      :fact_name => FactoryBot.create(:fact_name, :name => 'operatingsystem'))
-    @fact_trend = FactoryBot.create(:fact_trend, :trendable_type => 'FactName')
-    @fact_trend_valid_attrs = { :trendable_type => 'FactName', :trendable_id => @fact_os.fact_name_id.to_s }
-  end
-
-  test "should get index" do
```

Comment:
Should we maintain a test that makes sure the proper message is returned if the plugin isn't installed?

Reply:
It's a good idea, but I'm not sure it deserves the effort, I'll try though

Reply:
It was quite easy, done that ^_^